app/style/ChangePassphraseModal.less

```diff
@@ -14,6 +14,7 @@
 .change-passphrase-modal-section {
     padding: 20px;
     background-color: #fff;
+    box-shadow: @modal-dropshadow;
```

Comment:
[Mixins](http://lesscss.org/features/#features-overview-feature-mixins) might be a better way to handle this specific case, because you could do something like this:

    .mixin-modal: {
        box-shadow: 0px 13px 21px rgba(0, 0, 0, 0.21);
    }
    
    .change-passphrase-modal-section {
        .mixin-modal;
        /* other rules */
    }

And if in the future we want to customize all modal dialogs in some additional way, we would just add to the `mixin-modal` class.